### DISCUSSION TITLE
Decentralized-Entertainment

### DISCUSSION THREAD

**Discussion by LameChad** [naturalcrit/homebrewery#2075] [Ideas]
I'm a young entrepreneur and writer; and I love homebrewery

I have an idea for something special and I wanted to reach out to you guys before hiring a developer that doesn't know/care about the community. If this isn't the right channel to contact ya'll, I assume someone will correct me. 

I have an idea for a collaborative and competitive content creation platform. Someone would publish a piece of content, then another person can edit it- which instantiates an editable copy with reference back to the original. This would be repeated to create an iterative, fractal chain of variations on an original piece; all building on one-another to create something better than the original, measured by popularity amongst consumers by either purchase amount or hopefully for this a simple upvote button to keep it free.

I'm not sure 'blockchain' quite fits. I think decentralized-entertainment is a good term to sum it up. 

I'm reaching about because I could see this being a huge hit on homebrewery; and because, selfishly, I'd love to rip the architecture for use on the site I'm dreaming up; which I'm sure is as selfish and one-sided as it sounds. 

If you're interested, I'd love to discuss partnering up. 

LameChad

**Comment by Gazook89** [comment]
I'm going to close this discussion because it's over a year old with no response.  To give my personal response to it, for completeness sake, I am not interested because I have limited time (and skill, to be honest) and not looking to make a publishing platform, only a tool.  Also, even though it says "not blockchain", the mere mention of the word in such a post turns me off.

If you've made any headway on this or have a refined vision for it, re-open this discussion or post on the subreddit.

**Comment by calculuschild** [comment]
I'll echo @Gazook89 here; I don't think the Homebrewery is quite the right platform for this; we want to focus on making a good document editor rather than this.

>... I'd love to rip the architecture for use on the site I'm dreaming up; which I'm sure is as selfish and one-sided as it sounds.

However, this project is open source. If you still have interest in your idea, by all means go ahead; you can use our code as a starting point and make the changes you want as needed.

**Comment by ericscheid** [comment]
> I have an idea for a collaborative and competitive content creation platform. Someone would publish a piece of content, then another person can edit it- which instantiates an editable copy with reference back to the original. This would be repeated to create an iterative, fractal chain of variations on an original piece; all building on one-another to create something better than the original, measured by popularity amongst consumers by either purchase amount or hopefully for this a simple upvote button to keep it free.

Additionally, I would suggest @LameChad take a look at FedWiki.

- http://fed.wiki.org/view/federated-wiki
- https://github.com/fedwiki
